### PR TITLE
fix: python example server back online

### PR DESCRIPTION
**Pull request by william-silversmith**
Several issues fixed:
- missing invocation of ./node_modules/webpack/bin/webpack.js in package.json scripts
- incorrect invocation of URL rather than file path in example.py
- passing None to file_open, resulting in no file handle being available (unable to call None)
- many appearently missing modules from package.json

Added:
- block to server.py that allows for invocation of example.py as `python example.py` rather than `python -i example.py`

### CODE REVIEW

File: python/example.py [hunk 1]
```diff
@@ -23,5 +26,8 @@
 #   voxel_size=[6, 6, 40], 
 #   graph='./snemi3d/snemi3d_graph.pickle'
 # )
+url = viewer.get_viewer_url()
+webbrowser.open(url)
+print(url)
 
-webbrowser.open(viewer.get_viewer_url())
+# neuroglancer.block()
```
Comment (wongwill86) [inline python/example.py]: why is your fancy new block function commented out?

File: python/neuroglancer/static/__init__.py [hunk 2]
```diff
@@ -77,6 +77,9 @@ class FileSource(StaticContentSource):
     def __init__(self, path=None, file_open=open):
         if path is None:
             path = os.path.join(os.path.dirname(__file__), '../../../dist/dev')
+
+        file_open = file_open or open
```
Comment (wongwill86) [inline python/neuroglancer/static/__init__.py]: what was this for?

Comment (william-silversmith) [inline python/neuroglancer/static/__init__.py]: None was being passed to file_open, so when it went to open the file, it died. :(

File: package.json [hunk 3]
```diff
@@ -13,9 +13,9 @@
   "scripts": {
     "generate-code": "node ./config/generate_code.js",
     "build-min": "node --max_old_space_size=2048 ./node_modules/webpack/bin/webpack.js --config ./config/webpack.min.js --display-modules",
-    "build": "node --max_old_space_size=2048 --config ./config/webpack.config.js --display-modules",
-    "build-key_name_logger": "node --max_old_space_size=2048 --config ./config/webpack.key_name_logger.js",
-    "build:watch": "node --max_old_space_size=2048 --config ./config/webpack.config.js --watch",
+    "build": "node --max_old_space_size=2048 ./node_modules/webpack/bin/webpack.js --config ./config/webpack.config.js --display-modules",
```
Comment (william-silversmith) [inline package.json]: it was looking for the thing that was taking --config, which is not node

**Comment by william-silversmith** [comment]
I added some logic to make the example.py server accept ctrl-C in all situations and allow you to use `-i` when necessary. Problem solved. :D

**Comment by william-silversmith** [comment]
Rebased on Nico's master requirements.txt updates.

**Comment by wongwill86** [comment]
lgtm! make sure to create an issue re: source maps in production

**Comment by william-silversmith** [comment]
Issue created! https://github.com/seung-lab/neuroglancer/issues/174

**Pull request merged**